cirq/google/sim/xmon_simulator.py

```diff
@@ -53,6 +54,37 @@
 from cirq.study import ParamResolver, Sweep, Sweepable, TrialResult
 
 
+def pretty_state(wvf, decimals=2):
+    """
+    Returns the wavefunction as a string in Dirac notation.
+
+    For example:
+    circuit = cirq.Circuit.from_ops(cirq.H(Q1))
+    simulator = cirq.google.XmonSimulator()
+    result = simulator.simulate(circuit)
+
+    print(wavefunction(result.final_state)) -> -0.71|0⟩ + -0.71|1⟩
```

Comment:
The doc now describes the result and not what this method does.  I'd just give an example where you pass in a simple wave function and pretty print it.

Reply:
Oops! Forgot to update doc! Done now.